docs/getting-started/raster/auth/raster_authentication.md

```diff
@@ -8,7 +8,7 @@ On web clients, when using a Map Component such as Cesium for example, you will
 ```javascript
 const wmts = new Cesium.WebMapTileServiceImageryProvider({
   url: new Cesium.Resource({
-    url: "<RASTER-RASTER-SERVING-SERVICE_URL>/wmts/<layer_name>/{TileMatrixSet}/{TileMatrix}/{TileCol}/{TileRow}.png",
+    url: "<RASTER-SERVING-SERVICE_URL>/wmts/<layer_name>/{TileMatrixSet}/{TileMatrix}/{TileCol}/{TileRow}.png",
```

Comment:
it doesn't have to be `.png` since having JPEG, we should indicate that for the different image formats

Reply:
should be 'copy template from WMTS capabilities'

Reply:
fixed, changed into <LAYER_FORMAT>